languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -30590,6 +30590,13 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Groß- und Kleinschreibung -->
     <!-- ====================================================================== -->
     <category id="CASING" name="Groß-/Kleinschreibung">
+        <rule id="AHNUNG_KLEIN" name="keine ahnung (Ahnung)" default="temp_off">
+            <pattern case_sensitive="yes">
+                <token>ahnung</token>
+            </pattern>
+            <message>Das Nomen <suggestion><match no="1" case_conversion="startupper" /></suggestion> wird großgeschrieben.</message>
+            <example correction="Ahnung">Ich habe keine <marker>ahnung</marker>.</example>
+        </rule>
```

Comment:
Könnte auch ein Vertipper von "Mahnung" sein. -> Spelling

Reply:
Der Speller findet den Fehler schon, hat da eine Regel wirklich genug Mehrwert (außer, dass man weniger Vorschläge bekommt und die Message besser ist)?

Reply:
Ok, ja ich dachte das wäre unwahrscheinlicher, aber stimmt. Dann ist es wohl besser, "ahnung" weiterhin dem Speller zu überlassen...

Reply:
Oder besser: Ich habe die Regel um Mahnung erweitert. Die anderen Vorschläge sind mE nicht wirklich sinnvoll bzw. wahrscheinlich.
![image](https://github.com/languagetool-org/languagetool/assets/115984740/ef050b33-3c75-4bf6-8e2f-1f6e419d8d1b)

Reply:
@danielnaber ich denke, "keine Ahnung" wird häufig flüchtig in Chats oder Kurznachrichten geschrieben und da wäre es vlt. besser nur die eine Korrektur anzubieten bzw. jetzt _Ahnung_ und _Mahnung_.
Kann aber gerne auch wieder rückgängig gemacht und dem Speller überlassen werden.